var/spack/repos/builtin/packages/root/package.py

```diff
@@ -309,6 +309,9 @@ class Root(CMakePackage):
         conflicts('+' + pkg, when='@6.18.00:',
                   msg='Obsolete option +{0} selected.'.format(pkg))
 
+    # Feature removed in 6.26.00:
+    conflicts('+vmc', when='@6.26:', msg="VMC was removed in ROOT v6.26.00.")
+
```

Comment:
You can just add a `when=` argument to the variant e.g. `when=@:6.25`

Reply:
Hi @alalazo, thanks for the suggestion - I'd prefer to merge the changes as is and update the recipe to fix all of the variants that have been removed/change over time ('memstat', 'qt4', 'table') in a future PR

Reply:
Error messages are currently still :hankey:, so yes it's expected.